src/pages/projectSettings/tabs/GeneralTab/getFormSchema.tsx

```diff
@@ -342,7 +351,8 @@ export const getFormSchema = (
         repotrackerDisabled: {
           "ui:widget": widgets.RadioBoxWidget,
           "ui:description":
-            "Repotracker will be triggered from GitHub push events sent via webhook.",
+            "Repotracker will be triggered from GitHub push events sent via webhook. " +
+            "The repotracker create mainline builds for merged commits.",
```

Comment:
Using string addition shouldn't be necessary... if ESLint rejects this you could use string literals (`) instead
```suggestion
            "Repotracker will be triggered from GitHub push events sent via webhook. Repotracker creates mainline builds for merged commits.",
```